packages/clerk-js/src/ui/UserProfile/AddAuthenticatorApp.tsx

```diff
@@ -40,10 +40,12 @@ export const AddAuthenticatorApp = (props: AddAuthenticatorAppProps) => {
   return (
     <ContentPage.Root headerTitle={title}>
       {!totp && (
-        <Spinner
-          colorScheme='primary'
-          size='lg'
-        />
+        <Flex center>
```

Comment:
🔧 This deserves to be a primitive, shared across all screens that use it.

Reply:
I am always confused between primitives and elements. :)

Reply:
> I am always confused between primitives and elements. :)

We will soon extract the styled-system and the primitives to a separate package, I hope this will help avoid the confusion in the future :)